planner/core/find_best_task.go

```diff
@@ -478,6 +587,107 @@ END:
 	return bestTask, cntPlan, nil
 }
 
+// findBestTask implements LogicalPlan interface.
+func (p *LogicalSequence) findBestTask(prop *property.PhysicalProperty, planCounter *PlanCounterTp, opt *physicalOptimizeOp) (bestTask task, cntPlan int64, err error) {
```

Comment:
What's the difference between `(p *baseLogicalPlan) findBestTask` and `(p *LogicalSequence) findBestTask`?

Reply:
+1